ddr/ddr/Models/GameEntities/BatteryBar.swift

```diff
@@ -37,6 +37,13 @@ class BatteryBar: SKSpriteNode {
     }
   }
   
+  func addOne() {
```

Comment:
you should probably directly tie the display to the battery value instead of changing it explicitly